tests/training/test_utils.py

```diff
@@ -47,82 +47,22 @@ def setUp(self):
 
     def test_lcwa_margin_ranking_loss_helper(self):
         """Test if output is correct for the LCWA training loop use case."""
-        factory = TriplesFactory.from_labeled_triples(triples=self.triples)
```

Comment:
@mberr this test was failing because of the interface change - I think we could either delete it or just move it as an explicitit test in the losses tests